explainerdashboard/to_html.py

```diff
@@ -129,47 +124,10 @@ def card(html:str, title:str=None, subtitle:str=None, border=True)->str:
 </div>
 """
 
-def dashboard_card(title:str=None, description:str=None, url:str=None)->str:
-    """Generate a dashboard description car for ExplainerHub. 
-    Consists of title, description and url.
-    """
-    return f"""
-<div class="card">
-  <div class="card-header">
-    <h3 class="card-title">{title}</h3>
-  </div>
-  <div class="card-body">
-    <div class="w-100">
-    {description}
-    </div>
-  </div>
-  <div class="card-footer">
-    <div class="w-100">
-    <a href="{url}" class="stretched-link">Go to dashboard</a>
-    </div>
-  </div>
-</div>
-"""
-
-
-def card_deck(*cards)->str:
```

Comment:
this is no longer supported in bootstrap 5 I guess, so need to do something along these lines:

https://stackoverflow.com/questions/64311368/bootstrap-5-alpha-2-card-deck-example-not-showing-like-it-is-in-documentation

Reply:
I'll need to configure the card widths a bit more, most will need a simple `width=6` or 12 to fill half or the full space.

Reply:
yeah, but idea is that this should be automatic and responsive. So you can't just hard code it at 6 or 12. Just not specifying the width of the cols should do the trick though I guess (bootstrap will divide the cols evenly).